shor/gates.py

```diff
@@ -13,16 +13,49 @@
 class _Gate(_Layer):
     """Abstract base quantum gate class
 
-    # Properties
     input_length = valid length of input qubits
     qubits = indices of qubits, to be used as input to gate.
```

Comment:
I think we should be consistent.

I prefer "qbits" over "qubits" but not sure if that will confuse people.

Can find out in user testing.
Another option is the alias approach.
qbits = qubits = ...

Reply:
Can find and replace `qubits` -> `qbits`, or we could leave this alone for now. And just ask this question in user testing.